app/grandchallenge/reader_studies/views.py

```diff
@@ -395,7 +397,7 @@ class ReaderStudyDisplaySetList(CivSetListView):
         *CivSetListView.columns,
     ]
 
-    default_sort_column = 3
+    default_sort_column = 2
```

Comment:
Shouldn't this be 4 now?

Reply:
Oh yeah, order makes a lot more sense to keep instead of ID.